lib/twilio/audiohelper.ts

```diff
@@ -382,6 +407,16 @@ class AudioHelper extends EventEmitter {
     }
   }
 
+  /**
+   * Remove event listener for microphone permissions
+   * @private
+   */
+  _stopMicrophonePermissionListener(): void {
```

Comment:
```suggestion
  /**
   * Remove event listener for microphone permissions
   */
  private _stopMicrophonePermissionListener(): void {
```
You can set this to private (instead of internal public) since it's not used outside of this file.